clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -769,11 +769,12 @@ private void handleListOffsetResponse(Map<TopicPartition, Long> timestampsToSear
                 for (LogEntry entry : partition.records.entries()) {
                     for (LogRecord record : entry) {
                         // Skip the messages earlier than current position.
```

Comment:
Probably better to drop this comment. It is now inaccurate, and also unnecessary in general.